pkg/network/tls.go

```diff
@@ -33,7 +33,7 @@ func LoadServerTLSConfigFromFile() (*tls.Config, error) {
 		Certificates: []tls.Certificate{cert},
 		ClientCAs:    certPool,
 		ClientAuth:   tls.RequireAndVerifyClientCert,
-		ServerName:   certificates.FakeDnsName,
+		ServerName:   "knative-myns",
```

Comment:
Should we check the old name _as well as_ the new name?

Reply:
My assumption is that we are on the path to getting to namespace SANs, we keep the old names just to ensure we get there in gradual non disruptive steps...